cockatrice/src/dlg_forgotpassword.cpp

```diff
@@ -0,0 +1,67 @@
+#include <QLabel>
+#include <QGridLayout>
+#include <QHBoxLayout>
+#include <QDialogButtonBox>
+#include <QMessageBox>
+#include <QDebug>
+
+#include "dlg_forgotpassword.h"
+
+DlgForgotPassword::DlgForgotPassword(QWidget *parent)
+    : QDialog(parent)
+{
```

Comment:
Sure, but I still think this should be something we add in. Once we merge something like this, it might be a while until we touch it again